### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
It looks good, so merging this.

File: utils/source_code_examples.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    end do
+
+    ! print '(a)', 'P2'
+    ! print '(i0, 1x, i0)', i_max, j_max
+    ! print '(i0)', 255
+    ! do j = 1, 3
+    !   do i = 1, 3
+    !     print '(i0)', image(i,j)
+    !   end do
+    ! end do
```
Comment (certik) [inline utils/source_code_examples.js:59]: You reverted to an older version of the code. Can you please update it?

Comment (certik) [reply utils/source_code_examples.js:59]: I don't understand what happened -- this PR says merged, but the commit in master is https://github.com/lfortran/lcompilers_frontend/commit/2a6b874893d2aa1862b14f603e558115b51980aa, which is the correct (updated) version.

Comment (Shaikh-Ubaid) [reply utils/source_code_examples.js:59]: >You reverted to an older version of the code.

Sorry, by mistake, I committed the previous version of the code. 

>Can you please update it?

I later (very soon) observed it and so rebased that particular commit and pushed into master. 

>I don't understand what happened -- this PR says merged, but the commit in master is https://github.com/lfortran/lcompilers_frontend/commit/2a6b874893d2aa1862b14f603e558115b51980aa, which is the correct (updated) version.

Yes, it was rebased. I guess this `PR` was already merged and therefore did not get updated.

I apologize for all this confusion, Ondrej. I will be careful next time.

Comment (certik) [reply utils/source_code_examples.js:59]: Ah, no worries then, thanks for fixing it!

**Pull request merged**